tcf_website/templates/reviews/delete_confirm_modal.html

```diff
@@ -0,0 +1,24 @@
+<!-- Delete Confirmation Modal -->
+<div class="modal fade" id="deleteReviewModal{{review_id}}" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Delete Review</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Are you sure you want to delete this review?
+
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">No</button>
+        <form method="POST" action="{% url 'delete_review' review_id %}">
```

Comment:
I was able to delete someone else's review by changing this `review_id` manually in the browser developer tools.
![image](https://user-images.githubusercontent.com/7676354/100984594-45f14500-358e-11eb-85fe-c3e70a50785d.png)
I know no one's gonna do this, but for security reasons, can we check if the author of the review matches the one who clicked it? The CSRF token didn't prevent me from doing this.

Reply:
oh hmmmmmm I'll try to do it in the form/view

Reply:
![image](https://user-images.githubusercontent.com/4724192/101140260-b759e380-35e0-11eb-811a-547911a8459a.png)
![image](https://user-images.githubusercontent.com/4724192/101140220-aad58b00-35e0-11eb-9eb6-987da0aef4af.png)
Checking for the user & raising a permission denied error (redirects to http403)
Hmm we should probably create custom 404 and 403 error pages